package/yast2-storage-ng.spec

```diff
@@ -68,6 +68,15 @@ This package contains the files for YaST2 that handle access to disk
 devices during installation and on an installed system.
 This YaST2 module uses libstorage-ng.
 
+%package -n yast2-partitioner-ng
```

Comment:
I would remove the "-ng" part here. There was never a yast2-partitioner package in the past, so no need to distinguish it. Not everything relying on storage-ng has to receive the "-ng" suffix. :-)